src/client/components/Input/Input.css

```diff
@@ -0,0 +1,34 @@
+input {
```

Comment:
This input component has only one job to take care of the input where ever it is used but you wrote three classes "small", "medium" and "large" with the same styling. Please remove the extra CSS and try never to restrict the height/width of the elements like input boxes and buttons they inherit the size from their main container.
And since this app is only using only one font so you don't need to mention it this many times, it will inherit it that as well from the Container css.